validphys2/src/validphys/process_options.py

```diff
@@ -190,6 +190,16 @@ def _hqp_ptq_xq2map(kin_info):
     return Q / kin_info[_Vars.sqrts], Q * Q
 
 
+def _hqp_mqq_xq2map(kin_info):
+    # Compute x, Q2
+    QQMASS2 = (2 * TMASS) * (2 * TMASS)
```

Comment:
Since in many of the datasets you also have `m_t2` please use that instead of `TMASS`.